changelog/_unreleased/2022-11-09-fix-setting-null-data-while-expecting-array-on-a-storefront-response.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Fix setting null-data while expecting array on a StorefrontResponse
+issue: NEXT-24082
+author: Stephan Niewerth
+author_email: snw@heise.de
+author_github: stephanniewerth
+---
+# Storefront
+* `src/Storefront/Framework/Routing/StorefrontResponse.php` return an empty `array` on `getData()` if `$data` is `null`
```

Comment:
Changelog entries have to begin with certain keywords e.g. `Changed`, `Deprecated` or `Added` please reword your entries, so they start with those keywords